CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- used openers within the library only exposes `get_data` and `fake_data` methods due to breaking changes within
+  substra-tools
+- a third argument `task_properties` (containing the `rank` of a task) has been added to all algo methods relying on substra-tools
```

Comment:
can you move this to a `### Tests` section? Or at least say in the change that it only impacts the tests?

Reply:
done